doc/source/guidelines/index.rst

```diff
@@ -16,5 +16,13 @@ functionalities such as logging, data transfer...
 
    dev_practices
    app_interface_abstraction
-   service_abstraction
    data_transfer_and_representation
+   logging
+   service_abstraction
+
+
+List of guidelines
+-------------------
+
+* :ref:`ref_guide_logging`_ . Guidelines for logging in PyAnsys.
+  
```

Comment:
@akaszynski Let me know if you want to keep a list of all the guidelines.
If no, I will remove the ``ref:ref_guide_logging`` from both the `index.rst`and the` logging.rst` files.

Reply:
In general the more links the better. Let's keep this.

Reply:
I’ve replaced this with a table of contents.